FizzBuzz 1/FizzBuzz/Program.cs

```diff
@@ -1,32 +1,54 @@
 ﻿using System;
-using System.Text;
 
 namespace FizzBuzz
 {
     class Program
     {
         static void Main(string[] args)
         {
-            for (int i = 1; i <= 100; i++)
+            for (int i = 1; i <= 260; i++)
             {
-                StringBuilder output = new StringBuilder(16);
+                String[] outputArr = new String[5];
```

Comment:
Sorry, one small thing I missed before, you can just declare this variable with `var` instead of the type. Same when you declare your output string later 🙂